test/e2e/webdriver/chrome.js

```diff
@@ -37,6 +39,30 @@ class ChromeDriver {
     ];
 
     if (process.env.MULTIPROVIDER) {
+      // for multi-provider tests, we need to install the second extension
+      // since we assign the constant key to the extension, we need to generate a new random key (id) for the second extension
+      // this can be done by removing the key from the second extension's manifest and then installing the extension
+      const chrome2ManifestPath = path.join(
+        process.cwd(),
+        'dist',
+        'chrome2',
+        'manifest.json',
+      );
+
+      if (fs.existsSync(chrome2ManifestPath)) {
+        const manifest = JSON.parse(
+          fs.readFileSync(chrome2ManifestPath, 'utf8'),
+        );
+
+        // Remove the key from the manifest to generate a new random key (id) for the second extension
+        delete manifest.key;
+
+        fs.writeFileSync(
+          chrome2ManifestPath,
+          JSON.stringify(manifest, null, 2),
+        );
+      }
+
```

Comment:
There's a way to do this in way fewer lines of code.

In `test/e2e/set-manifest-flags.ts` we are already reading and writing to the manifest file.  On line 49, add

```
  if (process.env.MULTIPROVIDER) {
    delete manifest.key;
  }
```

then fix the type error you're going to get.  Voila!